_scripts/customJS/tokenParser.js

```diff
@@ -486,58 +503,50 @@ class TokenParser {
                 break;
             // end complicated options //
 
-            // REFACTOR OPTIONS //
-            // these might need to be refactored but I don't really understand the affiliation code yet
-            case "partof":
-                // currently we just use the affliation manager here to get a fully formatted string
-                value = AffiliationManager.getAffiliationPartOf(metadata, token.format)
-                formatter = "none"
-                break;
-            // END REFACTOR OPTIONS //
-
             default:
                 // if no special processing, check to see if it is a key in metadata, or failing that, displayDefaults
                 value = (this.#getParameterCaseInsensitive(metadata, token.token) ?? this.#getParameterCaseInsensitive(displayDefaults, token.token)) ?? ""
                 // default formatter is casing
                 formatter = "casing"
         }
 
-        if ((value || value === 0) && (!Array.isArray(value) || value.length > 0)) {
-
-            // apply casing format only to prefix and suffix //
-            // uses format, not firstFormat, which might not be ideal //
-            // could wrap in a function and pass either format or firstFormat //
-
-            let casingFormat = token.format ?? ""
-            casingFormat = casingFormat.split('').filter(char => this.casingChars.includes(char)).join('');
-            if (casingFormat) {
-                token.prefix = this.#getFormattedCaseString(token.prefix, { format: casingFormat })
-                token.suffix = this.#getFormattedCaseString(token.suffix, { format: casingFormat })
-            }
+        // if we don't have a value, return
```

Comment:
This is just to reduce nested and because it is a bit easier to read as "things that count as nothing" rather than "these things count as not nothing"

Reply:
makes sense